service/chat.py

```
@@ -150,3 +150,15 @@ async def nocontext_completions(conversation_body:str, model_dict, api_version:s
         usage = Usage(completion_tokens = 1, prompt_tokens = 1, total_tokens = 1)
     )
     return chat_response
+
+def embeddings(model_dict, api_version:str = None):
+    print("Entry Embeddings")
+    print(model_dict)    
+    print(api_version)
+    system_fingerprint = os.getenv('system_fingerprint')
```

Ruff

Local variable `system_fingerprint` is assigned to but never used

Remove assignment to unused variable `system_fingerprint`

(F841)